etlhelper/setup_oracle_client.py

```diff
@@ -169,14 +178,16 @@ def _download_zipfile(zip_download_source):
         zip_download_source
     returns:
         path to downloaded zipfile
-
-    Fails gracefully if download fails. Sys exit -1 with helpful msg
```

Comment:
I think "fails gracefully...etc" can be left in, just needs the "Sys exit" bit removed

Reply:
I think this needs approval from @dvalters to resolve.  I think it's OK to remove the whole line.